lms/static/js/verify_student/incourse_reverify.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Set up the in-course reverification page.
+ *
+ * This loads data from the DOM's "data-*" attributes
+ * and uses these to initialize the top-level views
+ * on the page.
+ */
+ var edx = edx || {};
+
+ (function( $, _ ) {
+    'use strict';
+    var errorView,
+        el = $('#incourse-reverify-container');
+
+    edx.verify_student = edx.verify_student || {};
+
+    errorView = new edx.verify_student.ErrorView({
+        el: $('#error-container')
+    });
+
+    return new edx.verify_student.InCourseReverifyView({
+        courseKey: el.data('course-key'),
+        checkpointName: el.data('checkpoint-name'),
+        errorModel: errorView.model
+    }).render();
```

Comment:
I generally include the render call as the last line of a View's initialize function. There may be cases where this is not desired, but it InCourseReverifyView is rendered every time it is initialized I think it makes sense to move it there.